numba/tests/test_withlifting.py

```diff
@@ -280,12 +280,8 @@ def test_liftcall4(self):
     @unittest.skipIf(PYVERSION <= (3, 8),
                      "unsupported on py3.8 and before")
     def test_liftcall5(self):
-        with self.assertRaises(errors.CompilerError) as raises:
-            njit(liftcall5)()
-        # Make sure we can detect a break-within-with and have a reasonable
-        # error.
-        msg = ("unsupported control flow: with-context contains branches")
-        self.assertIn(msg, str(raises.exception))
+        # checks that the for-with-break construct 
+        njit(liftcall5)()
```

Comment:
stdout needs capturing for this call else `A` is printed to it!

Reply:
done